api.py

```diff
@@ -0,0 +1,85 @@
+from fastapi import FastAPI
+from fastapi.responses import StreamingResponse, HTMLResponse
+import cv2
+import time
+from camera_manager import CameraManger
+
+# Create api
+app = FastAPI()
+
+# Create camera manager
+camera_manager = CameraManger()
+
+@app.get("/")
+async def root():
+    """
+    Just for testing
+    """
+    return HTMLResponse("""
+    <html>
+        <h1>Help</h1>
+    </html>
+    """, status_code=200)
```

Comment:
This seems temporary. Consider directing the user to any known camera streams.

Reply:
I went ahead and deleted it.